### ISSUE TITLE
Feature request - spriter pro

### ISSUE THREAD

**Issue by K4stor** [pixijs/pixi-spine#81]
Hi

a similar tool to spine is spriter pro.
Do you think you could add support for that too ?

Regards

**Comment by ivanpopelyshev** [comment]
Its not my priority right now. Does it have FFD (mesh transforms) ?

**Comment by K4stor** [comment]
as it seems someone coded a JS lib for it. But not with pixie :D
https://github.com/flyover/spriter.js/blob/master/spriter.js

**Comment by ivanpopelyshev** [comment]
Wow, it costs only 12$ in Russia/CIS! Yeah, it will be good, I can implement it next month, after I finish with conversion of rpgmaker MV to pixi-v4.

**Comment by K4stor** [comment]
awesome!

**Comment by sjlynch** [comment]
Was any progress made on this? I find the $300 price on Spine to be completely unreasonable and it never seems to go on sale. Spriter looks like it has most of the same features (including some good looking FFD tools https://www.youtube.com/watch?v=V0ehOfGP5XY) for $60.

**Comment by ivanpopelyshev** [comment]
Hey, @sjlynch! Do you mean that one https://www.patreon.com/user/posts?u=2384552 ?

If you need it like right now, you can mash pixi-spine but with official spriter js runtime (if there is one)

**Comment by sjlynch** [comment]
@ivanpopelyshev Yup, that's the one. I know $3 monthly towards the pixijs patreon probably isn't a lot of motivation so I could put a $15 bounty on this feature too, as long as it gets merged into pixijs. I think Spriter support would be overall great for pixijs. I don't need this feature soon though, I can wait a few months.

**Comment by englercj** [comment]
Hey @sjlynch, thanks for being a Patreon; every dollar helps us run the pixi services so I really appreciate it. Spriter sounds like a great new feature to have, hopefully someone can find the time to get something working for you.

I'm unfortunately strapped at work right now, but I should have more time next year. Good luck!

**Comment by goldenratio** [comment]
bumping this thread :)
btw Spriter Pro is available for 1USD now in humble bundle https://www.humblebundle.com/game-developer-software-bundle

**Comment by cursedcoder** [comment]
well, it's not relevant to this repository, but to PIXI

anyway let's keep this thread and see if anybody wants to implement plugin for spriter

**Comment by CarsonRoscoe** [comment]
Hey just bumping to see if any progress has been made in this feature :)

**Comment by ivanpopelyshev** [comment]
We dont have enough time for spine issues , i dont think anyone has for spriter. Can you do it? :)

**Comment by djmisterjon** [comment]
hey guys you also have a plugin here from rmmv
https://github.com/AlexanderKanakis/RPGMV/blob/master/Spriter.js
It can give you good base to implement this in pixijs.

**Comment by ivanpopelyshev** [comment]
Lol, it has pixi-layers in it built-in :)

**Comment by ishaan-puniani** [comment]
Would be great to see the support officially. 
I found an extension for the same purpose https://github.com/brinsleylogic/pixi-spriter that worked on recently.

**Comment by ivanpopelyshev** [comment]
Put a bounty on it?
